src/OpenSearch/Client.php

```diff
@@ -2155,7 +2155,7 @@ public function request(
         string $method,
         string $uri,
         array $attributes = []
-    ): array|string|null {
+    ): \Traversable|array|string|null {
```

Comment:
Should this just be `iterable` instead of `\Traversable|array`?

Reply:
yep I agree, iterable is probably cleaner